Sources/PerformanceMetrics/Trackers/PerformanceMetrics+MultiTracker.swift

```diff
@@ -0,0 +1,142 @@
+import Foundation
+
+public extension PerformanceMetrics {
+
+    /// A performance metrics tracker that forwards performance measuring events to multiple trackers, while not doing
+    /// any tracking on its own.
+    public class MultiTracker: PerformanceMetricsTracker {
+
+        /// The configured sub trackers.
+        public let trackers: [PerformanceMetricsTracker]
+
+        /// The tracker's tokenizer.
+        private let tokenizer = Tokenizer<Tag>()
+
+        /// The tracker's token dictionary, containing the mapping between internal and sub trackers' tokens.
+        private let tokens = Atomic<[Token<Tag> : [Token<Tag>]]>([:])
+
+        /// Creates a new performance metrics multi trcker instance, with the specified sub trackers.
+        ///
+        /// - Parameters:
+        ///   -trackers: The sub trackers to forward performance measuring events to.
+        public init(trackers: [PerformanceMetricsTracker]) {
+            assert(trackers.isEmpty == false, "🙅‍♂️: trackers shouldn't be empty, since it renders this tracker useless!")
+
+            self.trackers = trackers
+        }
+
+        // MARK: - PerformanceMetricsTracker
+
+        /// Starts measuring the execution time of a specific code block identified by a particular identifier, by
+        /// propagating it to all registered sub tracker.
+        ///
+        /// - Parameters:
+        ///   - identifier: The metric's identifier, to group multiple metrics on the provider.
+        /// - Returns: A token identifying this particular metric instance.
+        public func start(with identifier: Identifier) -> Token<Tag> {
+
+            let token = tokenizer.next
+            let subTokens = startTrackers(with: identifier)
+
+            tokens.modify { $0[token] = subTokens }
+
+            return token
+        }
+
+        /// Stops measuring the execution of a specific code block while attaching any additional metric metadata, by
+        /// propagating it to all registered sub trackers.
+        ///
+        /// - Parameters:
+        ///   - token: The metric's identifying token, returned by `start(with:)`.
+        ///   - metadata: The metric's metadata dictionary.
+        public func stop(with token: Token<Tag>, metadata: Metadata? = nil) {
+
+            guard let subTokens = tokens.modify({ $0.removeValue(forKey: token) }) else {
+                assertionFailure("🔥: missing sub tokens for token \(token)!")
```

Comment:
Maybe, we can conform these log messages to the latest styleguide?

Reply:
Sure, will do 👍 

I also have to open a PR on our style guide one of these days 😅